libraries/shared/storage/diff_repository.go

```diff
@@ -54,9 +54,9 @@ func (repository diffRepository) CreateStorageDiff(rawDiff types.RawDiff) (int64
 }
 
 func (repository diffRepository) CreateBackFilledStorageValue(rawDiff types.RawDiff) error {
-	_, err := repository.db.Exec(`SELECT * FROM public.create_back_filled_diff($1, $2, $3, $4, $5)`,
+	_, err := repository.db.Exec(`SELECT * FROM public.create_back_filled_diff($1, $2, $3, $4, $5, $6)`,
 		rawDiff.BlockHeight, rawDiff.BlockHash.Bytes(), rawDiff.HashedAddress.Bytes(),
-		rawDiff.StorageKey.Bytes(), rawDiff.StorageValue.Bytes())
+		rawDiff.StorageKey.Bytes(), rawDiff.StorageValue.Bytes(), repository.db.NodeID)
```

Comment:
Maybe this should pull the NodeID off of RawDiff (actually I'm pretty positive it should). 

If so, is repository.db.NodeID correct when doing a backfill?